docs/content/ingestion/schema-changes.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: doc_page
+---
+# Schema Changes
+
+Schemas for datasources can change at any time and Druid supports different schemas among segments.
+
+## Replacing Segments
+
+Druid uniquely 
+identifies segments using the datasource, interval, version, and partition number. The partition number is only present if 
+there are multiple segments created for some granularity of time. For example, if you have hourly segments, but you 
+have more data in an hour than a single segment can hold, you can create multiple segments for the same hour. These segments will share 
+the same datasource, interval, and version, but have linearly increasing partition numbers.
+
+```
+foo_2015-01-01/2015-01-02_v1_0
+foo_2015-01-01/2015-01-02_v1_1
+foo_2015-01-01/2015-01-02_v1_2
+```
+
+If at some later point in time, you reindex the data with a new schema, the newly created segments will have a higher version id.
+
+```
+foo_2015-01-01/2015-01-02_v2_0
+foo_2015-01-01/2015-01-02_v2_1
+foo_2015-01-01/2015-01-02_v2_2
+```
+
+Druid supports atomically replacing segments on a datasource and granularity level. In our example, once all `v2` segments are 
```

Comment:
I think this part until the end of the section might be clearer as,

> Druid batch indexing (either Hadoop-based or IndexTask-based) guarantees atomic updates on an interval-by-interval basis. In our example, until all `v2` segments are loaded in a Druid cluster, queries exclusively use `v1` segments. Once all `v2` segments are loaded and queryable, all queries ignore `v1` segments and switch to the `v2` segments. Shortly afterwards, the `v1` segments are unloaded from the cluster.
>
> Note that updates that span multiple segment intervals are only atomic within each interval. They are not atomic across the entire update.

Maybe?